index.html

```diff
@@ -66,5 +72,236 @@ <h3>
         </li>
       </ul>
     </section>
+
+    <!-- Work section -->
+    <div>
+      <works class="works">
+        <cards class="cards">
+          <block class="block1">
+            <img class="card-img" src="/images/Snapshoot.png" alt="" />
+          </block>
+          <block class="block2">
+            <div class="box1">
+              <h1 class="title1">Tonic</h1>
+              <ul class="frame1">
+                <li class="A Li">
+                  <i class="bi bi-circle-fill icon"></i> CANOPY
+                </li>
+                <li class="B Li">
+                  <i class="bi bi-circle-fill icon"></i> Back End dev
+                </li>
+                <li class="C Li">
+                  <i class="bi bi-circle-fill icon"></i> 2019
+                </li>
+              </ul>
+            </div>
+
+            <div class="box2">
+              <p class="card-para">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="box2-ul">
+                <li class="box2-li">HTML</li>
+                <li class="box2-li">Css</li>
+                <li class="box2-li">JavaScript</li>
+              </ul>
+              <button class="big-btn">See project</button>
```

Comment:
you have done a good job on styling this section only buttons here should have the same interaction design as in the template your button has a different onPress and hover text and background colors

![temp1 student onclick](https://user-images.githubusercontent.com/59707859/227724077-f6d0953c-3a78-421e-a96f-cb1d95d1cbd9.PNG)

![temp1 button event](https://user-images.githubusercontent.com/59707859/227724149-4c054ba7-86cd-45be-92d2-88abf39733c3.PNG)